engine/launcher/src/test/scala/org/enso/launcher/NativeLauncherSpec.scala

```diff
@@ -0,0 +1,18 @@
+package org.enso.launcher
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import buildinfo.Info
+
+class NativeLauncherSpec extends AnyWordSpec with Matchers with NativeTest {
```

Comment:
`AnyWordSpec` and `Matchers` should be rolled into `NativeTest`. This is what we do with `CompilerTest` and `InterpreterTest`.